geopandas/sindex.py

```diff
@@ -481,6 +538,73 @@ def query_bulk(self, geometry, predicate=None, sort=False):
                 input_geometry_index.extend([i] * len(res))
             return np.vstack([input_geometry_index, tree_index])
 
+        def nearest(self, coordinates, num_results=1, objects=False):
+            """
+            Returns the nearest object or objects to the given coordinates.
+
+            Requires rtree, and passes parameters directly to
+            rtree.index.Index.nearest.
+
+            This behaviour is deprecated and will be updated to be consistent
+            with the pygeos PyGEOSSTRTreeIndex in a future release.
+
+            If longer-term compatibility is required, use rtree.index.Index.nearest
+            instead - this could be accessed by calling
+            super(type(s.sindex), s.sindex).nearest()
```

Comment:
I am not sure if we should recommend this. Longer term, we might want to stop inheriting, and rather use composition (to not have those inheritance issues of conflicting behaviour).  
If people really want the rtree behaviour, then they can use rtree directly?